model/Core/Classes/Regulation.md

```diff
@@ -0,0 +1,28 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Regulation
+
+## Summary
+
+Regulation represents a rule or directive maintained by an authority.
+
+## Description
+
+The Regulation class represents any rules or directives enforced by an official authority.
+Regulations within this class can be either mandatory or voluntary.
+
+Examples of these regulations include the EU Radio Equipment Directive, EU Cyber Resilience Act,
+the Japan Cyber STAR, or the U.S. Cyber Trust Mark.
+
+## Metadata
+
+- name: Regulation
+- SubclassOf: Element
+- Instantiability: Concrete
+
+## Properties
+
+- mandatory
+  - Range: xsd:boolean
```

Comment:
```suggestion
  - type: xsd:boolean
```

This should be "type".
See https://github.com/spdx/spdx-3-model/blob/develop/docs/format.md#classes